### PR TITLE
[Example] use local image in all demos

### PR DESCRIPTION
**Pull request by gusye1234**
Signed-off-by: Jianbai Ye <jianbaiye@outlook.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,13 +3,13 @@
 
 openvino-mobilenet-image/mobilenet.bin
 openvino-mobilenet-image/mobilenet.xml
-openvino-mobilenet-image/input.jpg
+# openvino-mobilenet-image/input.jpg
```
Comment (hydai) [inline .gitignore:6]: Please remove these lines directly.

**Review by hydai — CHANGES_REQUESTED** [review]
What's the license of these photos? Is it acceptable to put them into our source tree?

**Comment by gusye1234** [comment]
The license for those two images is [here](https://github.com/bytecodealliance/wasi-nn/blob/main/rust/examples/images/LICENSE), which is the same as this repo

**Comment by hydai** [comment]
> The license for those two images is [here](https://github.com/bytecodealliance/wasi-nn/blob/main/rust/examples/images/LICENSE), which is the same as this repo

Putting a link to identify the source of these photos is better.

**Comment by hydai** [comment]
Hi @gusye1234 
Could you add a link to the original image? I will merge this once the link and license information is done.

**Review by hydai — APPROVED** [review]

**Comment by gusye1234** [comment]
Add the links and licenses in readme

**Comment by hydai** [comment]
Thanks!

**Pull request merged**